packaging/suse/velum.spec.in

```diff
@@ -0,0 +1,123 @@
+#
+# spec file for package velum
+#
+# Copyright (c) __CURRENT_YEAR__ SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+Name:           velum
+
+# When you release a new version, set Version and branch accordingly.
+# For example:
+# Version:      1.0.0
+# %define branch 1.0.0
+
+Version:        __VERSION__
+%define branch __BRANCH__
+Release:        0.0.1
+License:        Apache-2.0
+Summary:        Dashboard for CasP
```

Comment:
oops, yeah the description is still too short anyways... i forgot to update that